app/views/admin/users/_user_results.html.haml

```diff
@@ -23,4 +25,9 @@
   = paginate @users
 
 - else
-  %p= "No users matched that search"
+  - if params[:search].present?
+    %p No users matched your search.
+  - else
+    %p
+      There are no users,
+      = link_to "click here to create a new user.", new_path_for_user
```

Comment:
`Perfect` is a big word, especially if there is literally not a single qualified UX/Interaction Designer at TFG. All we have is Google, no actual research.